Corgibytes.Freshli.Cli/Functionality/Configuration.cs

```diff
@@ -0,0 +1,16 @@
+using System.IO;
+using Corgibytes.Freshli.Cli.Functionality;
+
+namespace Corgibytes.Freshli.Cli.Functionality;
+
+public class Configuration : IConfiguration
+{
+    public string GitPath { get; set; }
+    public string CacheDir { get; set; }
+
+    public Configuration(IEnvironment environment)
+    {
+        GitPath = "git";
+        CacheDir = Path.Combine(environment.HomeDirectory, ".freshli");
+    }
```

Comment:
This means we have default values in two places. We probably want to use these values in `AnalyzeCommand` as well.

Reply:
That's a good point about default values being in two places. I'll take a look at that.

Reply:
This should be addressed in 6984249.

Reply:
I found another one. Fixed in 573f68b.